demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -211,11 +211,12 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
           try {
             chimeMeetingId = await this.authenticate();
           } catch (error) {
+            const mediaDeviceInfoErrorMessage = 'getUserMedia is not allowed in HTTP sites. Either use HTTPS or enable media capture on insecure sites.';
             (document.getElementById(
               'failed-meeting'
             ) as HTMLDivElement).innerText = `Meeting ID: ${this.meeting}`;
             (document.getElementById('failed-meeting-error') as HTMLDivElement).innerText =
-              error.message;
+              error.message.includes('MediaDeviceInfo is not defined') ? mediaDeviceInfoErrorMessage : error.message;
```

Comment:
Its possible that a few browsers do not support the MediaDeviceInfo API like the Opera, which is a Chromium based browser. Ref. https://developer.mozilla.org/en-US/docs/Web/API/MediaDeviceInfo

The error might still be 'MediaDeviceInfo is not defined'. So with the current 'mediaDeviceInfoErrorMessage' it might confuse the user even after the user redirected to HTTPS in a non supported browser as the user will continue to see the same error.

How about using the following instead 
```
const mediaDeviceInfoErrorMessage = 'MediaDeviceInfo is not defined. Please redirect to HTTPS or use one of the officially supported browsers';
 ```

Reply:
I'm changed the implementation to only show the new error message when error occurs and the user is using `http:` protocol.